jest.config.ts

```diff
@@ -0,0 +1,28 @@
+import nextJest from 'next/jest';
```

Comment:
since you can implement react-testing-library standalone, why decide to use jest within?

Reply:
I added Jest separately since I wanted to test a few utility functions using Jest alone without having to use React Testing Library, since I didn't need to render any components to test those functions